tests/test_xopen.py

```diff
@@ -724,3 +724,20 @@ def test_valid_compression_levels(writer, level, tmpdir):
     with writer(test_file, "wb", level) as handle:
         handle.write(b"test")
     assert gzip.decompress(Path(test_file).read_bytes()) == b"test"
+
+
+def test_format_override(tmpdir):
```

Comment:
It’s a bit inconsistent at the moment, but I’d like all tests to be converted to use `tmp_path` instead of `tmpdir` at some point. It would be great to use `tmp_path` in new code.

Reply:
Agreed. Done.